package/upgrade/upgrade_node.sh

```diff
@@ -586,13 +586,30 @@ command_single_node_upgrade() {
   # Add logging related kube-audit policy file
   patch_logging_event_audit
 
+  # wait all fleet bundles
+  wait_for_fleet_bundles
+  echo "scale down fleet-agent"
+  kubectl scale --replicas=0 deployment/fleet-agent -n cattle-fleet-local-system
+  kubectl rollout status deployment fleet-agent -n cattle-fleet-local-system
+
   # Upgarde RKE2
   upgrade_rke2
+
   wait_rke2_upgrade
   clean_rke2_archives
 
   convert_nodenetwork_to_vlanconfig
 
+  # when RKE2 is upgraded ready, the new API-server is ready
+  sleep 10
```

Comment:
before `upgrade_os`, there are PVC based PODs which cannot start, the `wait_for_fleet_bundles` stuck in loop, needs further work

maybe @bk201 's solution to scale up the `fleet-agent` in upgrade controller can work.

```
+ kubectl get bundle.fleet.cattle.io -A
NAMESPACE     NAME                                          BUNDLEDEPLOYMENTS-READY   STATUS
..                    
fleet-local   mcc-rancher-monitoring                        0/1                       NotReady(1) [Cluster fleet-local/local]; deployment.apps cattle-monitoring-system/rancher-monitoring-grafana [progressing] Deployment does not have minimum availability., Available: 0/1


kk get pods -A 
NAMESPACE                   NAME                                                      READY   STATUS              RESTARTS       AGE

cattle-monitoring-system    alertmanager-rancher-monitoring-alertmanager-0            0/2     ContainerCreating   0              13m
cattle-monitoring-system    prometheus-rancher-monitoring-prometheus-0                3/3     Running             0              13m
cattle-monitoring-system    rancher-monitoring-grafana-8559b68898-s64zq               0/3     Init:0/2            0              13m
```